libs/community/langchain_community/vectorstores/lancedb.py

```diff
@@ -371,10 +371,12 @@ def _query(
 
         prefilter = kwargs.get("prefilter", False)
         query_type = kwargs.get("query_type", "vector")
+        metrics = kwargs.get("metrics", "l2")
```

Comment:
Double-checking that:

1. This is the current default;
2. We want to maintain a separate default in Langchain.

Otherwise, may be preferable to do
```python
if metrics := kwargs.get("metrics"):
    lance_query = ...
else:
    ...
```

Reply:
1. Thanks @multiduplikator for showing that 'l2' is the current default
2. I then suggest that we leave the default value up to lance and implement the fix as proposed by you. I've updated the PR